README.md

```diff
@@ -62,3 +63,8 @@ Step 7
 * Node.js & Express
 * It's necessary to write the testing for both frontend and backend
 * Deploy your application to `CI` like `Travis`, `Circle` or any else.
+
+## Shell command
+* npm start		Run & compile the project and show on the Internet
```

Comment:
Please highlight your code block by **`** (backticks).

Also, please remove the unnecessary guildline of the assignment and write down yours.

ref: [document-style-guide
](https://github.com/ruanyf/document-style-guide/blob/master/docs/structure.md)